benchto-driver/src/main/java/com/teradata/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -164,12 +164,15 @@ public void setup()
     {
         LOGGER.info("Searching for benchmarks in classpath ...");
 
-        List<Path> benchmarkFiles = Files
-                .walk(properties.benchmarksFilesPath())
-                .filter(file -> isRegularFile(file) && file.toString().endsWith(BENCHMARK_FILE_SUFFIX))
-                .collect(toList());
-        benchmarkFiles.stream().forEach((path) -> LOGGER.info("Benchmark found: {}", path.toString()));
+        ImmutableList.Builder<Path> benchmarkFilesBuilder = ImmutableList.builder();
+        for (Path benchmarkFilesPath : properties.benchmarksFilesDirs()) {
+            Files.walk(benchmarkFilesPath)
```

Comment:
can't do, walk might return `IOException` within stream. `IOException` is perfectly legal for: `findBenchmarkFiles`